scripts/analytics/preprint_summary.py

```diff
@@ -81,7 +82,7 @@ def get_class():
     args = preprint_summary.parse_args()
     yesterday = args.yesterday
     if yesterday:
-        date = (datetime.today() - timedelta(1)).date()
+        date = (timezone.now() - timedelta(1)).date()
```

Comment:
It's a little better to use named arguments here, since timedelta includes a lot of optional parameters, and having (day=1) is a tiny bit more readable. Yes, not your code originally, but since we're here.